concepts/nil/about.md

```diff
@@ -1,30 +1,67 @@
-# Zero Values
+# Introduction
 
-Unlike some other languages, Go has no empty/null/undefined state for uninitialized variables. Every type has a [zero value][zero_values] that uninitialized variables will hold upon declaration.
+Go does not have a concept of undefined for variable values. Variables declared without an explicit initial value default to the [zero value][zero_values] for their respective type.
```

Comment:
```suggestion
Go does not have a concept of empty/null/undefined for variable values. Variables declared without an explicit initial value default to the [zero value][zero_values] for their respective type.
```

I'd like to keep this bit in - it's useful for people from coming from other languages to understand there is no `null` (as `null` and `undefined` are quite different things)

Reply:
Done!